### ISSUE TITLE
Can we refer to our lab codes and the prof's lecture codes for the finals?

### ISSUE THREAD

**Issue by kenjimo2** [nus-cs2030/2122-s1#357]
Also for the finals, can we use a software like Intellij to test our answers before submitting?

**Comment by JustinS00** [comment]
Yup we are able to refer to our lab codes and prof lecture notes. My TA told us using jshell should be fine but not sure about using Intellij tho

**Comment by JadeHZY** [comment]
> Yup we are able to refer to our lab codes and prof lecture notes. My TA told us using jshell should be fine but not sure about using Intellij tho

According to my TA, using IntelliJ is also fine!
